src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -108,72 +108,69 @@ export default ajaxCaller(class ShareWorkspaceModal extends Component {
     }, [
       h(IdContainer, [id => h(Fragment, [
         h(FormLabel, { htmlFor: id }, ['User email']),
-        h(AutocompleteTextInput, {
+        h(CreatableSelect, {
           id,
+          isLoading: !loaded,
           placeholder: 'Add people or groups',
-          value: searchValue,
-          onChange: v => this.setState({ searchValue: v }),
-          suggestions: _.difference(suggestions, _.map('email', acl)),
-          style: { fontSize: 16 }
+          'aria-label': 'Enter an email address of a person or group',
+          isValidNewOption: val => !validate.single(val, { email: true, exclusion: aclEmails }),
```

Comment:
There is a bug that crashes the app in some cases when entering/validating email addresses